src/bin/juliaup.rs

```diff
@@ -109,6 +116,11 @@ enum SelfSubCmd {
     #[cfg(feature = "selfupdate")]
     /// Uninstall this version of juliaup from the system
     Uninstall {},
+    #[cfg(not(feature = "selfupdate"))]
+    /// If `selfupdate` feature is not enabled, this
+    /// configures the Uninstall to just print an error message
+    /// that it is **unavailable**.
```

Comment:
I believe the text after the tripple `/` will be output as the help text in the compiled version, so it should address end-users, not devs looking at the source code :) So maybe just `Uninstall this version of juliaup from the system (UNAVAILABLE)` or something like that?

Reply:
Oh i didnt know the comment is used for the help text. Resolved